package.json

```diff
@@ -146,6 +146,7 @@
     "find-up": "4.1.0",
     "firebase": "7.14.5",
     "flat": "5.0.2",
+    "form-data": "4.0.0",
```

Comment:
You can remove this dependency since `globalThis.FormData` already exists in Node.js 18+